modules/custom/openy_analytics/src/AnalyticsCron.php

```diff
@@ -276,7 +276,8 @@ function getParagraphsUsage() {
     $statement->addExpression('COUNT(type)', 'count');
     $paragraphs_counted = $statement->execute()->fetchAllKeyed();
 
-    $bundles = array_keys($this->entityTypeBundleInfo->getBundleInfo('paragraph'));
+    // $bundles = array_keys($this->entityTypeBundleInfo->getBundleInfo('paragraph'));
+    $bundles = array_keys(\Drupal::service('entity_type.bundle.info')->getBundleInfo('paragraph'));
```

Comment:
We have to rebuild it according to DI.

Reply:
This is code adoption, not a refactoring. All refactorings should happen after we are on Drupal 9

Reply:
ok

Reply:
fixed https://github.com/ymcatwincities/openy/pull/2174/commits/839562bfa412cf71c9862ca9d9ce54b72f92ff06